plan-for-extension-exercise/src/planForExtensionExercise/Shape.java

```diff
@@ -0,0 +1,9 @@
+package planForExtensionExercise;
+
+public abstract class Shape {
```

Comment:
Sorry, I wasn't very clear. Please update this pull request to include the change, and close the other pull request.